examples/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/NYTimes/gizmo/examples
+
+replace github.com/NYTimes/gizmo => ../../gizmo
```

Comment:
might want to just make it `../` just in case someone has the name of the repository different from that default one.